components/builder-web/app/origin/origin-page/origin-integrations-tab/origin-integrations-tab.component.ts

```diff
@@ -36,12 +36,11 @@ export class OriginIntegrationsTabComponent implements OnInit, OnDestroy {
     private confirmDialog: MatDialog,
     private route: ActivatedRoute,
     private title: Title
-  ) { }
+  ) {}
 
   ngOnInit() {
-    this.store.dispatch(fetchIntegrations(this.origin.name, this.token));
-
     this.sub = this.route.parent.params.subscribe((params) => {
+      this.store.dispatch(fetchIntegrations(params.origin, this.token));
```

Comment:
This fixes the bug with navigating directly to this URL, which sometimes renders no results.